README.md

```diff
@@ -1,2 +1,36 @@
-# hm-content-toc
-WordPress plugin. Creates content TOC for specified header elements from the current post content. Adds anchor elements in the content just before the matched headers.
+[![Build Status](https://travis-ci.org/dashaluna/hm-content-toc.svg?branch=master)](https://travis-ci.org/dashaluna/hm-content-toc)
+
+=== Plugin Name ===
+Contributors: dashaluna, tcrsavage, johnbillion, sanchothefat
```

Comment:
@tcrsavage @johnbillion @sanchothefat let me know if you want me to delete your username from here :) Not sure what this affects..